packages/data-store-json/src/data-store-json.ts

```diff
@@ -137,6 +138,18 @@ export class DataStoreJson implements IAgentPlugin {
     }
   }
 
+  async dataStoreDeleteMessage(args: IDataStoreDeleteMessageArgs): Promise<boolean> {
```

Comment:
@mirceanis I see that you originally removed this function, can you review this file's changes?

Reply:
There were some unpleasant consequences when deleting stuff because of some typeorm misconfiguration in our entities, causing a cascade effect and deleting more than just the message, or throwing foreign key errors.

This should not be the case now, but this is the reason why some "delete" methods are weirdly missing or commented out.

The missing methods were not added to the data-store-json plugin to keep the API uniform, but we need to review this and normalize the API.